tests/python/gaia-ui-tests/gaiatest/apps/ring_tone/app.py

```diff
@@ -26,14 +29,25 @@ def tap_save(self):
         Wait(self.marionette).until(lambda m: save_button.get_attribute('disabled') == 'false')
         save_button.tap()
 
+    def tap_exit(self):
+        self.marionette.find_element(*self._header_locator).tap(25, 25)
```

Comment:
This has the same bug where it won't detect the back button: used your workaround